conclusion.tex

```diff
@@ -48,14 +51,17 @@ \section{Conclusion}
                     \textbf{Germany} & \textbf{Aggressive Reduction} & \textbf{23,868}\\
                     \hline
                 \end{tabularx}
-    \caption {\gls{EU} nations and their respective \gls{UNF} inventory. The bolded countries'
-              \gls{UNF} inventory adds up to the required \gls{UNF} amount for French \gls{SFR} transition. }
+    
     \label{tab:which_send}
 
 \end{table}
 
 In these simulations, some complex political and economic factors were not 
-incorporated and various assumptions were present in this scenario. However,
+incorporated and various assumptions were present in this scenario (For
```

Comment:
Parenthetical statements should generally be avoided if they can instead be a full sentence. This part should probably be "incorporated and various assumptions were present in this scenario. For example, Germany's current policy is to not reprocess its \gls{LWR} fuel \cite{topfer_germany_2011}, and this policy would create a shortage in the supply of \gls{LWR} \gls{UNF} for \gls{ASTRID} \gls{MOX} production."